src/MPDevice_win.cpp

```diff
@@ -71,31 +71,32 @@ QString MPDevice_win::getLastError(DWORD err)
     return result;
 }
 
-HANDLE MPDevice_win::openDevice(QString path)
+HANDLE MPDevice_win::openDevice(QString path, bool exlusive /* =false */)
 {
+    DWORD sharedOptRW = exlusive ? 0 : FILE_SHARE_READ | FILE_SHARE_WRITE;
     HANDLE h = CreateFileA(qToChar(path),
                            GENERIC_WRITE | GENERIC_READ,
-                           0, //Open exclusively
-                           NULL,
+                           sharedOptRW,
```

Comment:
same as before?

Reply:
Yes, opening device exclusivly, except during enumerate, when it is peaking the pid/vid.